docs/source/database.rst

```diff
@@ -2,14 +2,137 @@
+  a single value and can be iterated on, `first()` returns only the first value, etc.
+
+As an example of multiple filters, the code below:
+
+.. code-block:: python
+
+    records = MIRIMyMonitorStats.objects.filter(aperture__iexact=ap, mjd_end__gte=60000)
+    
+    filters = {
+        "aperture__iexact": ap,
+        "mjd_end__gte": 60000
+    }
+    records = MIRIMyMonitorStats.objects.filter(**filters)
+
+show two different ways of combining a search for a particular aperture *and* only data 
+taken more recently than MJD=60000.
+
+Note that django executes queries lazily, meaning that it will only actually *do* the 
+query when it needs the results. The above statement, for example, will not actually 
+run the query. Instead, it will be run when you operate on it, such as
+
+* Getting the length of the result with e.g. `len(records)`
+* Printing out any of the results
+* Asking for the value of one of the fields (e.g. `records[3].aperture`)
+
+Q Objects
+=========
+
+In order to make more complex queries, Django supplies "Q Objects", which are essentially 
+encapsulated filters which can be combined using logical operators. For more on this, see
+`the django Q object documentation <https://docs.djangoproject.com/en/5.0/topics/db/queries/#complex-lookups-with-q-objects>`_.
+
+Storing Data
+------------
+
+Django also uses the model tables (and objects) directly for storing new data. For example,
+if you have a monitor table defined as below:
+
+.. code-block:: python
+
+    from django.db import models
+    from django.contrib.postgres.fields import ArrayField
+
+    class NIRISSMyMonitorStats(models.Model):
+        aperture = models.CharField(blank=True, null=True)
+        mean = models.FloatField(blank=True, null=True)
+        median = models.FloatField(blank=True, null=True)
+        stddev = models.FloatField(blank=True, null=True)
+        counts = ArrayField(models.FloatField())
+        entry_date = models.DateTimeField(blank=True, null=True)
+
+        class Meta:
+            managed = True
+            db_table = 'niriss_my_monitor_stats'
+            unique_together = (('id', 'entry_date'),)
+            app_label = 'monitors'
+
+then you would create a new entry as follows:
+
+.. code-block:: python
+
+    values = {
+        "aperture": "my_aperture",
+        "mean": float(mean),
+        "median": float(median),
+        "stddev": float(stddev),
+        "counts": list(counts.astype(float)),
+        "entry_date": datetime.datetime.now()
+    }
+    
+    entry = NIRISSMyMonitorStats(**values)
+    entry.save()
+
+There are (as usual) a few things to note above:
+
+* Django doesn't have a built-in array data type, so you need to import it from the 
+  database-compatibility layers. The ArrayField takes, as a required argument, the type
+  of data that makes up the array.
+* In the Meta sub-class of the monitor class, the `all_label = 'monitors'` statement is 
```

Comment:
```suggestion
* In the Meta sub-class of the monitor class, the `app_label = 'monitors'` statement is 
```

Reply:
Change made. I realized that the right place to put this documentation was a different part of readthedocs, which is why it's marked as outdated.